tools/tertiary-analysis/scanpy/scanpy-regress-variable.xml

```diff
@@ -1,22 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<tool id="scanpy_regress_variable" name="Scanpy RegressOut" version="@TOOL_VERSION@+galaxy0">
+<tool id="scanpy_regress_variable" name="Scanpy RegressOut" version="@TOOL_VERSION@+galaxy1">
   <description>variables that might introduce batch effect</description>
   <macros>
     <import>scanpy_macros2.xml</import>
   </macros>
   <expand macro="requirements"/>
   <command detect_errors="exit_code"><![CDATA[
-ln -s '${input_obj_file}' input.h5 &&
-PYTHONIOENCODING=utf-8 scanpy-regress
-    --keys '${variable_keys}'
-    @INPUT_OPTS@
-    @OUTPUT_OPTS@
+#if $variable_keys
+  ln -s '${input_obj_file}' input.h5 &&
+  PYTHONIOENCODING=utf-8 scanpy-regress
+      --keys '${variable_keys}'
+      @INPUT_OPTS@
+      @OUTPUT_OPTS@
+#else
+  echo "No regression variables passed, simply passing original input as output unchanged.";
+  cp '${input_obj_file}' '${output_h5}'
```

Comment:
A naive question without knowledge of the inner workings of galaxy, guess rename or symlink is not sufficient here?